**Antibiotic induced alterations of the murine gut microbiota and
subsequent effects on colonization resistance against *Clostridium
difficile***

Alterations of the Murine Gut Microbiota that Lead to *Clostridium
difficile* Colonization

*Infection and Immunity* (IAI) is devoted to the advancement and
dissemination of fundamental knowledge concerning (i) the molecular
basis of infections caused by pathogenic bacteria, fungi, and parasites;
(ii) mechanisms of pathogenicity, including virulence factors (such as
toxins, adhesins, invasins, and secreted effector molecules) and other
microbial products that are or may be related to pathogenesis; (iii) how
these virulence factors interact with the host cell to alter cellular
processes; (iv) factors involved in host resistance and susceptibility
to infection; (v) immunity to pathogenic microorganisms, including the
functions of phagocytes, lymphocytes, immunoglobulins, and other
factors; (vi) experimental models of infection and the pathological
consequences of infection; and (vii) selected aspects of vaccines
against pathogens (see below).

IAI will not consider papers that are preliminary, purely descriptive,
or case studies. Clinical studies may be suitable for consideration by
IAI if they provide novel insights into infectious disease pathogenesis.
IAI welcomes studies of the microbiome that relate to host-pathogen
interactions. Papers describing methodology are not encouraged; only
under unusual circumstances will they be considered for publication.

**ISME:**
[**http://mts-isme.nature.com/cgi-bin/main.plex?form\_type=display\_auth\_instructions\#format**](#format)

**5000 words with abstract**

**Science?/PNAS?/Plos Biology**

**BMC journals**

**mBio**

Alyxandria M. Schubert,^1^ Hamide Sinani,^1^ Patrick D. Schloss^1^\*

^1^Department of Microbiology and Immunology, University of Michigan,
Ann Arbor, Michigan

\* Correspondence: pschloss@umich.edu

734.647.5801

Department of Microbiology and Immunology

University of Michigan

1520A Medical Science Research Building I

1150 W. Medical Center Dr.

Ann Arbor, MI 48109

**ABSTRACT**

**250 words**

**\
INTRODUCTION **

The microbiome, or the diverse community of microorganisms living in and
on the body, has an integral role in deterring pathogen colonization and
infection. This native protection by the microbiome from invasive
pathogenic species is termed colonization resistance. It is well
established that the gut bacterial microbiome is critical in the hosts’
defense against the pathogen *Clostridium difficile*. When this
indigenous community is perturbed, this often leads to a loss of
resistance. This is especially important in many hospital settings where
patients are not only exposed to various types and degrees of
perturbations, such as antibiotics, diet changes, chemotherapy, but also
exposed to the environmental *C. difficile* spores. Among
hospital-acquired infections in the United States, *C. difficile*
infections (CDI) are the number one reported and are responsible for
14,000 deaths a year. It is not completely understood how different
perturbations affect subsequent colonization resistance, nor do we have
a comprehensive view of the taxonomic changes associated with a loss in
colonization resistance.

Need for more knowledge about interactions between members of the
community. Know that fmts work [anna’s paper], but don’t know why,
except that you recover the diversity and are receiving an established
community already that works. Individual probiotics have not worked
extensively [], whereas studies find that combinations of bacteria often
work better than by themselves [lawley, pamer, stein], and it’s common
for bacteria to change their behavior in the context of other bacterial
species [Crost 2010, quorum sensing-biofilms, recognizing fellow
bacterial species vs killing others, synergistic effects]. Understanding
the scope of relationships between bacteria in a community is necessary
to learn about its response, in concert with its host, to perturbations
including antibiotic-induced, pathogen invasion, probiotics doses, and
diet changes, etc. The purest way of studying these bacterial
interactions is by observing them in nature (in the context of their
whole biome) [ecology citation?]. These observations can be used to
inform models and networks to predict future responses to perturbation
(stein, Gordon diet paper) and characterize interactions
(psoriasis/Schubert, sparCC paper).

The purpose of this investigation was to test the effects of an array of
perturbations on microbiome CR levels against *C. difficile*. We then
used this information to design microbiome-based models for estimating
pathogen colonization levels. Through the administration of various
antibiotic regimens, we altered the murine gut microbiota and observed
these changes using 16S rRNA sequencing. The resultant gut communities
were then challenged with *C. difficile* spores. Through correlation and
random forest analysis of the starting communities with the subsequent
level of *C. difficile* colonization, we built linear models to predict
*C. difficile* colonization levels based on a subset of the microbiota.
We used experimental data to test the ability of the model.

Though we are limited in a resolution at the strain and species level,
we have identified groups of related species with antagonistic
relationships with *C. difficile* colonization. We identified several
models with equally robust quality statistics and observed that across
the top models a subset of 5 species were consistently present. [EXPAND,
implications/importance]

**\
MATERIALS AND METHODS**

**Animal Care**

We used 5-8 week old C57Bl/6 mice for all our experiments. These mice
were housed in our SPF room in the animal facility at the University of
Michigan. All animal-related protocols and experiments were approved by
the University Committee on Use and Care of Animals at the University of
Michigan.

**Antibiotic administration**

Mice were administered one of seven different antibiotics, including
cefoperazone, vancomycin, metronidazole, streptomycin, ciprofloxacin,
ampicillin, and clindamycin. The route of administration depended on the
specific antibiotic. Cefoperazone (either 0.5mg/ml, 0.3mg/ml, or
0.1mg/ml), vancomycin (0.625mg/ml, 0.3mg/ml, or 0.1mg/ml), streptomycin
(5mg/ml, 0.5mg/ml, or 0.1mg/ml), metronidazole (0.5mg/ml), and
ampicillin (0.5mg/ml) were all administered in the mouse drinking water
for 5 days. Ciprofloxacin (10 mg/kg) was administered via oral gavage,
and clindamycin (10 mg/kg) was administered via IP injection. All
antibiotic treated mice were given 24 hours of recovery on regular
drinking water prior to *C. difficile* challenge.

***C. difficile* Preparation and Challenge**

In all our experiments we used *C. difficile* strain 630Δerm [cite dena
lyras]. The spores used were prepared from a single large batch made
following previously established protocols [cite paul?]. Spores were
quantified within the week prior to *C. difficile* challenge for each
experiment [established protocol I can cite?]. Spores were quantified by
boiling a small volume at 65°C for 20 minutes to kill off residual
vegetative cells. The volume was then diluted and plated on TCCFA
plates. These were incubated at 37°C anaerobically overnight. The next
day *C. difficile* colony forming units (CFU) were counted and the
concentration of spores in the working stock was back calculated. On the
day of challenge, *C. difficile* spores, at a final concentration of
10^3^, were administered to mice via oral gavage. Immediately following
this challenge, the remaining *C. difficile* inoculum was diluted in a
series and plated to confirm the correct dosage.

**Sample Collection and Plating **

Fecal samples were freshly collected for each mouse immediately prior to
the administration of any antibiotic, during any antibiotic recovery
days, and days 0 and 1 post *C. difficile* challenge. Each fecal sample
on day 1 was weighed and diluted under anaerobic conditions with
anaerobic PBS 1:10 based on pellet weight. A dilution series was then
plated onto TCCFA plates [cite plates] and incubated at 37°C
anaerobically overnight. The colony forming units (CFU) were counted and
calculated the subsequent day.

**DNA Extraction and Sequencing**

Total bacterial DNA was extracted from each untreated and day 0 stool
sample using the MOBIO PowerSoil®-htp 96 Well Soil DNA Isolation Kit.
Sequencing preparation and sequencing was carried out as previously
described (Kozich JJ et al. 2013). Briefly, the V4 region of the 16S
rRNA gene was amplified, purified, and pooled in equimolar
concentrations. These libraries were then mixed with Illumina-prepared
PhiX control libraries. Sequencing was performed using Illumina’s MiSeq
platform to produce paired 250-nt reads. Mock/error/error of run?

**Sequence Curation**

These sequences were curated using mothur as previously described
(Kozich JJ et al and Schloss PD et al). The full pipeline is also
demonstrated online at the mother website
(<http://www.mothur.org/wiki/MiSeq_SOP>). Sequences were binned into
operational taxonomic units (OTUs) using a 3% dissimilarity cutoff.
Taxonomic assignments were determined by using a naïve Bayesian
classifier with the Ribosomal Database Project (RDP) training set
(version 9) requiring an 80% bootstrap confidence score. In order to
avoid biases due to uneven sampling, samples were normalized to 1,625
sequences per samples [Schloss, reducing effects]. For those samples
with more than 1,625 sequences per samples, the range was \_\_, the mean
was \_\_, the median was \_\_, the median absolute deviation was \_\_.
All 16S rRNA gene sequence data and the MIMARKS table are available at
\_\_.

**Statistical Analysis & Modeling **

Using data from the original antibiotic treatments groups, OTUs were
selected for addition into the model following several criteria. First
all OTUs were present in abundance at a minimum of 1% in at least one or
more samples. Second, these bacteria were the top 5 OTUs with the
strongest positive correlation or among the top 10 OTUs with the
strongest negative correlation with subsequent *C. difficile*
colonization. Correlation analysis was performed using Spearman’s rank
based correlation coefficient, ρ, for non-normally distributed data.
Third, we used the random forest feature selection algorithm to
determine influential bacteria in predicting *C. difficile* levels. The
top 15 OTUs that were not unclassified and had the highest percent
increase in MSE (mean squared error) towards the random forest model.
Finally, candidates in our model were screened for strong correlations
(\>0.9) with each other using sparCC [reference]. The highest
correlation we observed was 0.84 between OTUs 6 and 7, which dropped off
to 0.71 for other OTU pairs. This screening process allowed us to narrow
our candidate list down to 19 OTUs for possible inclusion into the
linear model.

Next, we used the R package “leaps” in order to determine the top linear
models given our candidate list of OTUs as parameters in the model. We
set the maximum number of parameters in the model to 10, due to the size
of our data set (99 observations). Using the regsubsets() function, we
used the exhaustive method for determining the 3 best models for each
number of parameters in the model from 1 to 10. Based on these results,
we decided on a final model incorporating 5 OTUs. This model was used to
predict the outcome of *C. difficile* colonization based on the Day 0
communities from the titration dataset. All statistical analyses were
conducted using R version 3.1.2.

**\
RESULTS**

** All tabular data must be accompanied by either standard deviation
values or standard errors of the means. The number of replicate
determinations (or animals) used for making such calculations must also
be included. All statements concerning the significance of the
differences observed should be accompanied by probability values given
in parentheses. The statistical procedure used should be stated in
Materials and Methods. be sure to cite all figures and tables.**

**Levels of colonization resistance are associated with the initial
structure of the gut microbiota.** Using untreated mice as well as those
treated with seven representatives from six classes of antibiotics
(**Table 1**) we generated distinct and reproducible community
structures that were assessed for their resistance to *C. difficile*
colonization (**Figure 1**). 16S rRNA gene sequencing of the communities
sampled from the feces of the mice sampled at the time of *C. difficile*
inoculation revealed six distinct community structures (AMOVA,
p\<0.001). The community structures of the beta-lactam-treated mice
(i.e. cefoperazone and ampicillin) were not significantly different from
each other (AMOVA, p=0.37). In addition, the untreated and
ciprofloxacin-treated microbiota were not significantly different from
each other (AMOVA, p=0.12). Although both pairs of treatment groups were
not significantly different across the entire community, there did
appear to be differences among the low-abundance taxa (**Figure 1**).
All other comparisons between the community structures were
significantly different from each other (AMOVA, p\<0.001). Subsequently,
24 hours after inoculation *C. difficile* was undetectable in the
untreated mice or the ciprofloxacin-treated mice; however it had
successfully colonized the mice in all of the other treatment groups to
similarly high levels (10^7^-10^8^ CFU/g) (**Figure 1**). We calculated
the diversity of all of the communities using the Inverse Simpson
(**Supplemental Figure 1**) and Shannon indices (data not shown) and
observed a significant negative correlation with subsequent *C.
difficile* colonization (both p\<0.001). These results suggest that
multiple microbiota structures are capable of being colonized by *C.
difficile* and that diversity may be an important factor in resistance
to invasion.

**Correlation analysis reveals potentially protective bacteria.** To
identify bacterial taxa that could be associated with resistance or
susceptibility to *C. difficile*, we measured the correlation between
the relative abundance of each OTU on the day of inoculation with the
level of *C. difficile* colonization 24 hours later across all treatment
groups (**Figure 2**). OTUs with negative a correlation were associated
with providing resistance against *C. difficile* (N=XX) and those with a
positive correlation were associated with susceptibility (N=XX). Among
various bacterial families, three were consistently negatively
correlated with *C. difficile* colonization: Porphyromonadaceae
(ρ~average~= -0.57), Lachnospiraceae (ρ~average~= -0.39),
Ruminococcaceae (ρ~average~= -0.47). OTUs from one bacterial phylum, the
Proteobacteria, had a consistent strong positive relationship with *C.
difficile* colonization. These included OTUs associated with the
*Pseudomonas* (ρ= 0.31)*,* *Rhodobacter* (ρ= 0.31)*,* and *Escherichia*
(ρ= 0.56) genera. Other notable OTUs that were found at the extremes of
the distribution of correlation scores included those affiliated with
the genus *Lactobacillus* (order Lactobacillales; OTU23, ρ= -0.73),
*Alistipes* (order Bacteroidales; OTU20, ρ= -0.78), *Enterorhabdus*
(phylum Actinobacterium; OTU29, ρ= -0.70), and *Streptococcus* (order
Lactobacillales; OTU78, ρ= 0.36). Overall, these results suggest that
several larger groups of related OTUs, particularly within the
Firmicutes phylum, may participate in colonization resistance against
*C. difficile*, while broadly many members of the Proteobacteria phylum
may be involved in susceptibility.

**Smaller perturbations of the microbiota result in maintenance of CR.**
Because most of our original treatments resulted in similarly high
levels of *C. difficile,* we hypothesized that applying smaller
antibiotic perturbations to the microbiota would result in reduced
sensitivity to *C. difficile*. To test this, we titrated the doses of
cefoperazone, streptomycin, and vancomycin given to the mice
(**Supplemental Table 1)**. Colonization levels decreased in the mice
receiving lower doses of cefoperazone and streptomycin in support of our
hypothesis; while these levels were similar across mice receiving
vancomycin, regardless of the dose (**Figure** **3**). Next, we
hypothesized that the populations we previously identified as being
associated with protection against colonization would be elevated in the
mice with decreased colonization and those associated with
susceptibility would be less abundant.

The titrated antibiotic doses clearly resulted in distinct community
structures that were variations of the original dose (**Figure 3**). The
cefoperazone titrations resulted in a shift in dominance from the
Bacteroidetes to the Firmicutes phylum with increasing antibiotic dose.
Most notable was the increase in OTUs associated with resistance
including the Porphyromonadaceae and *Alistipes* and a reduction in OTUs
associated with susceptibility including members of the Proteobacteria.
The relative abundance of other OTUs associated with protection
including the Lactobacillus, Lachnospiraceae, and Ruminococcaceae did
not change as expected based on the initial correlation analysis;
however, these OTUs were present at levels similar to the untreated
mice. The streptomycin titrations did not result in profound shifts in
the overall community structure, but did yield increased resistance.
Rather, there were individual populations that were gained or lost
across the gradient. Notably, mice receiving the lowest dose retained
higher levels of *Alistipes* and Lachnospiraceae, which we previously
found to be associated with colonization resistance. The relative
abundances of the Porphyromonodaceae and Ruminococcaceae were unchanged,
but were present at similarly high levels to those found in the
untreated mice. Although the vancomycin titration yielded a number of
large changes in the abundance of individual populations, there was no
recovery of colonization resistance. The relative abundance of many of
the populations associated with resistance increased with a decreased
vancomycin dosage; however, with the exception of the *Lactobacillus*
populations, they did not reach the levels observed in the untreated
mice. In addition, all of the vancomycin-treated mice had high levels of
OTUs affiliated with the *Akkermansia* and Enterobacteriaceae. Together,
these results suggest that individual populations are not sufficient to
provide colonization resistance. Rather, resistance is likely a product
of the overall composition of the community.

We next wanted to determine whether associations we observed with the
seven antibiotics at a single dose were supported by the results we
observed in the titration experiments. Using only the new titration
groups, we again performed correlation analysis of species level OTUs at
inoculation with the *C. difficile* colonization levels the next day
(**Figure 4**). Among many of the taxa previously identified as having
significant relationships with *C. difficile*, we observed similar
trends across the new titration data set, most notably with the
titration treatment groups of cefoperazone and vancomycin.

Overall, cefoperazone titration treatments supported the correlation
results from the original data set. The most positively correlated OTUs
were *Pseudomonas* (OTU65), *E. coli* (OTU11), and *Clostridium*
(OTU61), while the top 8 most negatively correlated were all different
Porphyromonadaceae OTUs (OTU3, 4, 6, 8, 13, 2, 7, 12). Vancomycin
titration treatments also supported the significant correlations found
from the original data set. Among those with correlations smaller
than -0.80, 7 Porphyromonadaceae OTUs (OTUs 19, 12, 3, 4, 2, 1, 13),
Lachnospiraceae OTU39, and *Alistipes* OTU20 supported significant
results found in the original data. Similarly, *Escherichia* (OTU11, ρ=
0.84) had the strongest positive correlation with subsequent *C.
difficile* levels. In addition, the new vancomycin titration treatment
results showed that *Akkermansia* had a strong positive relationship
(ρ=0.77). The relative abundance of *Akkermansia*, a member of the
Verrucomicrobia phylum, bloomed significantly following treatment with
any of the 3 doses from \<5% to as much as 50% of the community (p=?,
**Figure 4**). Streptomycin titration treatment experiment results had
the most sign discrepancies for direction of relationship (positive or
negative) compared with the original antibiotic results [quantify?].

However, those discrepancies that were significant were found among the
strongest positively correlated OTUs. This included Bacteroidetes OTU16
(ρ=0.61) and *Bacteroides* OTU5 (ρ=0.72). *Akkermansia* OTU10 was also
strongly positively correlated (ρ=0.64). *E. coli* OTU11 was near absent
from the streptomycin treated communities (**Figure 4**), and thus we
did not observe a strong association with *C. difficile* colonization as
in most previous antibiotic treatments. Some of the strongest negative
correlations were among several Porphyromonadaceae (OTUs 6, 18, 13),
*Turicibacter* OTU17 (ρ=-0.70), and *Alistipes* OTU20 (ρ=-0.69). All of
the statistically significant negative correlations from the titration
results support findings from the original antibiotic treatments.

**Allowing recovery of identified groups restores colonization
resistance.** For each antibiotic treatment, a control group of mice
receiving antibiotics without inoculation of *C. difficile* was
maintained. The microbiota’s of animals treated with metronidazole
underwent a drastic shift in the microbiota structure with an additional
5 days of recovery. We observed changes in relative abundance of several
OTUs that we previously observed had strong relationships with
subsequent *C. difficile* colonization (**Figure 5**). Notably there
were increases in overall abundance of the levels of Porphyromonadaceae,
Ruminococcaceae, and Lachnospiraceae, and decreases in levels of
Enterobacteriaceae. Based on these changes, we hypothesized that
delaying *C. difficile* challenge to allow these potentially important
changes in microbiota structure to occur would lead to a decrease in
colonization. On Day 1 post challenge of these recovered mice, we
observed a significant decrease in *C. difficile* level of CFU/g feces
(p=?). These results strengthen the importance of Porphyromonadaceae,
Lachnospiraceae, Ruminococcaceae, and Enterobacteriaceae in setting up
the right environment of negatively and positively correlated bacteria
to affect the outcome of *C. difficile* colonization.

**Modeling to predict colonization outcomes.** Given our ability to make
general predictions of *C. difficile* colonization based on the
microbiota, we next designed a model to calculate expected colonization
levels given the relative abundance of a specific OTU subset. We
identified this subset using only the original data set to build the
initial model. Random forest and correlation analyses were used to
determine the best candidate list of OTUs for potential inclusion in the
model [Materials & Methods]. These 19 candidates are listed in
Supplemental Table 2. We allowed our model to include anywhere from 1 to
10 parameters, given our dataset included 99 observations. Using an
exhaustive search to build models, we determined the best 3 models for
each number of parameters. Supplemental Figure 2 shows the results of
the model building. Interestingly, across the models with highest
adjusted R^2^ values, there are 5 OTUs that are common to each, OTUs 6,
7, 20, 39, and 283. We used a model incorporating these 5 OTUs to
predict the level of *C. difficile* CFU/g feces for the titration data
set to estimate the performance of the model, and we achieved an
adjusted R^2^ of 0.65.

**\
DISCUSSION (90 lines last paper)**

We designed statistical models based on a set of varied antibiotic
perturbations of the microbiota and their respective CR levels against
*C. difficile*. These models incorporated all combinations of top
candidate species-level OTUs. We then tested our top model’s ability to
use new abundance information on a specific subset of the microbiota to
predict the subsequent *C. difficile* colonization level, which was
experimentally validated. These results greatly expand our knowledge of
varied perturbations and their subsequent effects on microbiota
structure and function (colonization resistance ability). Furthermore
modeling important/key bacterial subsets of the community we can build
models to assess CDI risk.

Through our perturbation experiments we identified a panel of bacteria
with either strong positive or negative relationships with subsequent
levels of *C. difficile* colonization. A large number of
Porphyromonadaceae, Lachnospiraceae, Ruminococcaceae, and Clostridia, as
well as *Alistipes* were found to have strong potential resistance
abilities against *C. difficile*. Conversely, several members of the
Proteobacteria phylum, including Pseudomonas, Streptococcus, and
especially *E. coli* have strong potential positive impacts for *C.
difficile* colonization. These results were supported by their strong
correlation values supported across several antibiotic regimens. They
are also supported through experiments in which the original doses of
cefoperazone, streptomycin, and (vancomycin) were titrated to alter the
overall abundances but not membership of the community from the starting
dose, as has been observed previously (Sekirov). These subtler
structural changes allowed for greater range of colonization levels of
*C. difficile* as well as further/fine-tuned support for our original
results. Furthermore, by allowing a known change to occur in important
bacterial populations within metronidazole treated mice, we were able to
reduce the level of colonization by *C. difficile*. Lastly we showed
their importance through modeling techniques that statistically confirm
these results.

Previous work characterizing the microbiome’s of human and mouse hosts
in the context of CDI provide glimpses into potentially protective and
alternatively susceptible community structures, i.e. which bacteria are
present and their abundance. Cross-sectional investigations have
characterized the microbiota of patients in the hospital with and
without a *C. difficile* infection (CDI) and compared to healthy
controls. These studies found patients with *C. difficile* tended to
have a reduction in numbers of Clostridiales Incertae Sedis XI (Vincent
et al), Lachnospiraceae, Ruminococcaceae, and *Bacteroides* (Schubert et
al). These studies however lack the baseline healthy state of the
microbiota, which preclude them from assessing risk of *C. difficile*.
Murine models of CDI have the advantage of observing the specific
changes that occur before and after controlled perturbations and the
introduction of *C. difficile.* Communities with low levels of CR were
associated with blooms in either Lactobacillaceae (cefoperazone-induced)
(Theriot, 2014, bassis, 2014), Proteobacteria (induced by either
clinda+5abx cocktail, clinda alone, or tigecycline) (Reeves, 2011,
Buffie, 2011, Bassis 2014). Lachnospiraceae is associated with
resistance to *C. difficile* infection (Reeves 2011) and has been
directly shown to decrease colonization through mono-colonization
studies in germfree mice (Reeves 2012).

Even fewer studies attempt to model these observational changes in the
microbiome following perturbation. Using the time series data gathered
from the *C. difficile* challenge of clindamycin-treated mice (Buffie
2011), a Lotka-Volterra dyamics-based framework was built which
incorporates both time and accounts for external perturbations (Stein
2013). They identified a subset of their network, containing Blautia,
Akkermansia, and Coprobacillus, as having a negative relationship with
*C. difficile*, while alternatively Enterococcus had a positive
relationship.

**maybe make a point somewhere about how some whole groups like
clostridia/Clostridiales/rumino&lachno, porphyros can appear to be
protective, while for positive correlations/bad bugs its more species
specific.. coincidence? **

-   How relates to other studies

    -   Antonopoulos et al-reproducible community dynamics

    -   Antunes et al (finlay)—abx treatment on intestinal metabalome

    -   Sekirov—antibiotic induced perturbations of intestinal
        microbiota

        -   “Change in the compostion alone of the intestinal microbiota
            (in the absence of significant changes in total numbers of
            intestinal microbes) prior to Salmonella serovar Typhimurium
            infection was sufficient to make mice more susceptible to
            Salmonella serovar Typhimurium, indicating that different
            subsets of the microbiota may play protective roles or
            enhance susceptibility to enteric infections.”

        -   Streptomycin and vancomycin doses were administered to mice
            prior to challenge with another gastrointestinal pathogen
            *Salmonella enterica* serovar Typhymurium (Sekirov 2008).
            This study showed \_\_\_. Changes in community composition
            were dose dependent.

        -   Strep increases bacteroidetes (CFB), vanc increases firms
            and then proteo (with increasing dose)

    -   Reeves/theriot/robinson—clindamycin/cef/cocktail/antbiotic
        administration alters the community structure

    -   Croswell—prolonge impact of abx on intestinal microbial ecology
        salmonella

    -   Rea- effect of broad and narrow spectrum antimicrobials

    -   Ferreira—intestinal microbiota plays a role in salmonella
        induced colitis

    -   Reeves with clone libraries and cef

    -   Theriot?—at least the metabolome paper—before and after with
        metabolites and cef, also does she look at anything
        taxonomically? yup

**Many of the relationships we observed in our study between specific
bacteria and *C. difficile* are supported in the literature. –see my
comments in this section**

Within the Porphyromonadaceae, only OTU505 had a significant positive
relationship with subsequent *C. difficile* colonization (ρ= +0.21),
which by blastn shares 99% identity with *Coprobacter fastidiosus.*
Similarly within the Lachnospiraceae family, only one OTU had a positive
relationship with *C. difficile* colonization (OTU174, ρ= +0.22). The
Lachnospiraceae group also contained an OTU that had an exceptionally
strong negative correlation (OTU39, ρ= +0.77). The Clostridia class was
largely composed of the order Clostridiales (n=6), of which only one had
a positive relationship with *C. difficile* colonization (OTU154,
ρ= +0.26).

Several broader groups of bacteria can work together (in some
combination) to provide CR …possible that proteobacteria in general are
bad …These results indicate that many related species within the
larger/broader bacterial groups may be able to serve in providing
colonization resistance against *C. difficile.*

Other taxons in Figure 3 had more species specific correlations. For
instance the Actinobacteria phylum included *Enterorhabdus* (OTU29,
ρ= -0.70), Coriobacteriaceae (OTU50, ρ= -0.50), *Bifodbacterium* (OTU22,
ρ= -0.34), *Actinomyces* (OTU58, ρ= +0.23). The Lactobacillales group is
divided into two negatively correlated *Lactobacillus* OTUs (OTUs 23,
21, ρ= -0.73, -0.40 respectively) and two positively correlated
*Streptococcus* OTUs (OTUs 78, 79). Similarly, the Bacteroidales group
included species at both extremes, for example, an *Alistipes* (OTU20,
ρ= -0.78) and a *Bacteroides* species (OTU44, ρ= +0.34). [END sent, all
the other bad bugs look more like random smaller level taxonomic
classification divisions, as in it’s a genus/species level effect type
deal]

**--functional redundancy? Across related large groups like porphyros?**

**--mostly Firmicutes—better competitors with cdiff? Because similar
nutrients resources? **

**Also functional redundancy with a highly diverse community… not just
about who possibly but also ab being diverse—emphasizes community as a
whole**

OTU39—Eubacterium ventriosum (94%), eubacterium rectale (93%), roseburia
intestinalis/hominis/faecis (93%) (positively correlated to differences
observed in discordant MZ twins for BMI—more abundant in higher BMI
siblings… these are all butyrate producers, also capable of degrading
fiber)

FOS—fructooligosaccharides produced by degredation of inulin or
polyfructose

**Inulin and fructo-oligosaccharides have divergent effects on colitis
and commensal microbiota in HLA-B27 transgenic rats.**

### Abstract

Modulation of intestinal microbiota by non-digestible carbohydrates may
reduce inflammation in inflammatory bowel disease (IBD). The aim of the
present study was to assess the effects of inulin and
fructo-oligosaccharides (FOS) on intestinal microbiota and colitis in
HLA-B27 transgenic rats, a well-validated rodent model for IBD. In this
study, 4-week-old rats were fed 8 g/kg body weight inulin or FOS for 12
weeks, or not. Faeces were collected at 4 and 16 weeks of age; and
caecal samples were collected at necropsy. The effects of inulin and FOS
on chronic intestinal inflammation were assessed using a gross gut
score, histology score and levels of mucosal IL-1β. Intestinal
microbiota were characterised by quantitative PCR and denaturing
gradient gel electrophoresis. Colitis was significantly reduced in all
FOS-fed rats compared to the control diet, whereas inulin decreased
chronic intestinal inflammation in only half the number of animals.
Quantitative analysis of caecal microbiota demonstrated that inulin
increased the numbers of total bacteria and the
Bacteroides-Prevotella-Porphyromonas group, FOS increased
bifidobacteria, and both fructans decreased Clostridium cluster XI. In
the faecal samples, both inulin and FOS decreased total bacteria,
Bacteroides-Prevotella-Porphyromonas group, and Clostridium clusters XI
and XIVa. FOS increased Bifidobacterium spp., and mediated a decrease of
gene copies of Enterobacteriaceae and Clostridium difficile toxin B in
faeces. SCFA concentrations in the faecal and caecal samples were
unaffected by the diets. In conclusion, FOS increased the abundance of
Bifidobacterium spp., whereas both fructans reduced Clostridium cluster
XI and C. difficile toxin gene expression, correlating with a reduction
of chronic intestinal inflammation.

Any interesting points about the results? Mechanisms? Competition—ask
matt if they are good? Proteobacteria could definitely play a role in
susceptibility—creating an environment fit for cdiff… what ways could
all those other bugs be making a bad environment for cdiff? Using up
bile acids/germinants.., with those bugs gone freeing up sugars, … maybe
connecting a lot more to some of caseys metabolome paper, and how mine
would fit that

-   Limitations

    -   No smaller intervals? Of cdiff levels, or changes in microbiota

    -   Porphyromonadaceae is more prevalent in mice… so might not be as
        relevant for humans

    -   Different strains of cdiff have different microbiota
        requirements to protect?

    -   Doesn’t get at function… ?

    -   Collect more data to continue to improve our microbiome modeling
        abilities,

-   Contribution to the field, positive spins

    -   PROBIOTICS

    -   Possible global effects? CR against other GI pathogens?

    -   clinical applications—PCR diagnostic for risk

The final paragraph should highlight the main conclusion(s), and provide
some indication of the direction future research should take. (for and I
and I mention host)

**-therapeutic: continuously supply probiotics while lowering
inflammation of the host (get host side back to normal state in tandem,
via drugs? diet)… so attack the issue of CDI from both the microbiota
end and host end… microbiota-host-pathogen triangle… …squeeze it out
(eg. clindamycin clearance? If compared pathology of host at diff days
along 10d for clindamycin vs cef/met/amp (these other ones that are high
in cdiff coloniz), is host or toxin diff?... why does clinda clear after
being CDhigh but no other ABX?)**

The human microbiome is complex; it has been observed that there is no
identifiable “core” microbiome that all humans share (Turnbaugh 2009).
It’s been observed that there is a high level of interindividual
variation among healthy individuals’ microbiomes (Costello 2009). Thus
the definition of a healthy microbiome is one that encompasses a wide
range of microbiota structures, each one evolved with their host. It’s
highly likely there is no one size fits all solution for the treatment
or prevention of CDI. Thus characterizing the relationships within the
microbiome are important in the context of a community. Need more
studies that look at the level of the community as a whole in CR over
the effect of single bugs. Examples of papers looking at
combinations/subsets of bacteria in CR

It makes sense that many bacteria would play a role in colonization
resistance and that it is more about the community composition as a
whole *given* that interindividual differences is so great. Two people
share no more than \_\_% of their microbiomes. They are unique to
individuals and yet all these different profiles of microbiotas are
capable of colonization resistance. People do not get *C. difficile*
infections without first being exposed to some major purturbation of
their microbiome, namely antibiotics. Thus many structures of the
microbiome, and many distinct bacterial populations must be capable of
contributing to resistance. Identifying/cataloguing a comprehensive
profile of those bacteria with potential roles in CR would be an initial
step towards personalized/individualized treatments with known mixtures
of protective bacteria.

Bacteria within a community setting may act differently in the context
of other bacterial species.

[using this time model for the metro… I wonder if it doesn’t work
because of the time difference in the amount of recovery time? I’m
picturing sort of an echo/ripple effect where at first the waves, or
changes, are really large and then over time that rate of change of the
community slows and the changes less pronounced/smaller. In this context
the metro communities were at a stage farther along in the echo, meaning
changes not happening as fast. Whereas the measurements we’ve taken and
are using to make predictions are from the community at a time when you
expect more/bigger/faster changes. So then the results are calculated
assuming one rate but actually in a different (slower).

By this image, the initial strength of the perturbation matters too.]

**--also somewhere, ecoli are typically on avg (\_\_% of the community,
typically wayyyy outnumbered… whats the avg relabund of all those CR
OTUs in an untreated mouse, how does that compare to the abundance of
those bacteria I’ve implicated in –CR) …point is that Proteobacteria are
usually outnumbered… don’t have a chance, gut is suited to bact/firm
(which not surprisingly make up the major players in CR).. I bet those
other outlier bad bugs are also in low abund in untreated mice (?). but
when the gut environment changes for w/e reason, can give those rarer
bad ones/proteo/esp e. coli a chance to take hold. They could just be
individually opportunistic or work synergistically with cdiff. If
individually opportunistic, it might be that they are facing pretty much
a common enemy which is the lach/rum/clos/porphs and end up having
similar techniques/niches as a result of that common enemy. Working
synergistically—eg. b. theta sialidase where it cleaves off resources
for cdiff.**

**\
ACKNOWLEDGEMENTS**

We would like to thank Dena Lyras for providing the 630Δerm strain used
in this study***.*** We would also like to thank Mary A. M. Rogers for
her assistance with statistical questions. This work was supported by
several grants from the National Institutes for Health 1R01GM099514
(PDS), R01HG005975 (PDS), U19AI090871 (VBY, PDS), and P30DK034933 (PDS,
VBY). The funding agencies had no role in study design, data collection
and analysis, decision to publish, or preparation of the manuscript.

**CONFLICT OF INTEREST STATEMENT**

All authors have no conflict of interest to report.

**\
REFERENCES**

***References***

Authors are responsible for the accuracy of the references. Published
articles as well as those in press (please state the name of the journal
and enclose a copy of the manuscript) may be included. In the text of
the manuscript, a reference should be cited by author and year of
publication eg (Bailey & Kowalchuk, 2006) and (Heidelberg et al, 1994)
and listed at the end of the paper in alphabetical order of first
author. References should be listed and journal titles abbreviated
according to the style used by Index Medicus, examples are given below.
All authors should be quoted for papers with up to six authors; for
papers with more than seven authors, the first six only should be
quoted, followed by *et al*.

**Journal article:**

Cho JC, Kim MW, Lee DH, Kim SJ. (1997). Response of bacterial
communities to changes in composition of extracellular organic carbon
from phytoplankton in Daechung reservoir (Korea). Arch Hydrobiol
138:559–576.

Lawley with his 6 bug combo, including \_

Pamer-nature paper- designed consortium of protective bugs including *C.
scindens* (Lachnospiraceae), *Barnesiella intestihominis*
(Porphyromonadaceae), *Pseudoflavonifractor capillosus (*Clostridiales),
*Blautia hansenii* (Lachnospiraceae)

Stein—Lotka Volterra dynamics reveal subnetwork of bugs implicated in
either susceptibility or protection, containing Blautia, Akkermansia,
and Coprobacillus, as having a negative relationship with *C.
difficile*, while alternatively Enterococcus had a positive
relationship.

Schubert—modeling CD status using human data

sekirov

**\
FIGURE LEGENDS**

**Supplemental Table 1. Titration amounts for each antibiotic.** The
number of animals per group (n) are shown for each antibiotic and dose.

**Figure 4. Levels of *C. difficile* present on Day 1 for each titration
treatment.** The amount of *C. difficile* was measured by CFU/gram of
fecal sample collected from mice treated with **A)** cefoperazone
titrations, **B)** streptomycin titrations, or **C)** vancomycin
titrations.

**Figure 5. Relative abundances for titration groups.** Microbiota on
Day 0.

**Figure 6. Heatmap comparing correlation analysis between the original
data set and titration data sets.** Species level OTUs (3% cutoff) found
on Day 0 were correlated with the level of *C. difficile* CFU/g feces on
Day 1 using Spearman’s rank based correlation coefficient. Only OTUs
with an average abundance of at least 0.1% across all original
treatments were considered in the correlation analysis. All correlations
shown in this graph are significant (p\<0.05). The lowest common
classification among bacterial species was used to group bacteria
shown….

**Figure 7. Differences in day 0s for metronidazole. –maybe better way
to show because it takes up a lot of space**

**FIGURES **

![](media/image1.pdf)

**Figure 1. Microbiome structures following diverse antibiotics. **

Currently is phylotype at the genera level (tx=1).

-Most significant correlations? RF using antibiotic categories for
topdose data (family-level)? The 19 candidate OTUs?

![](media/image2.pdf)

**Figure 2. Correlation analysis of bacterial species present on Day 0
with *C. difficile* levels on Day 1.** Species level OTUs (3% cutoff)
found on Day 0 were correlated with the level of *C. difficile* CFU/g
feces on Day 1 using Spearman’s rank based correlation coefficient. Only
OTUs with an average abundance of at least 0.05% across all original
treatments were considered in the correlation analysis. All correlations
shown in this graph are significant (p\<0.05). The lowest common
classification among bacterial species was used to group bacteria as
shown on the x-axis along with the number OTUs (n) belonging to that
classification.

-also add possibly “with cdiff CFU day 1”on the y and maybe an x label
that says “3% level otus on day 0”

vanc titration:

![](media/image3.pdf)

cef titration:

![](media/image5.pdf)

strep titration:

![](media/image7.pdf)

**Figure 3.** The titration graphs… add section for the cdiff levels so
don’t need another graph… fix lines somehow… ask about fixing the widths

![](media/image9.pdf)

**Figure 4. Heatmap comparing correlation analysis between the original
data set and titration data sets. **

**-** Also reorder this by classification and see what happens

![](media/image10.emf)

**Figure 5. Differences in day 0s for metronidazole. **

-colors

-selected taxa

[FIGURE]

**Figure 6. Model figure showing its performance **

**\
TABLES**

  ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  **Antibiotic**        **Dose **     **Administration**                            **Class **        **Mechanism **                                     **Primary Target **
                                                                                                                                                         
                                      **(Route, Time)**                                                                                                  
                                                                                                                                                         
  --------------------- ------------- --------------------------------------------- ----------------- -------------------------------------------------- ---------------------
  Ciprofloxacin (n=5)   10 mg/kg      Oral gavage, Day -1                           Fluoroquinolone   Inhibits DNA gyrase                                Gram +/-

  Clindamycin (n=11)    10 mg/kg      IP injection, Day -1                          Lincosamide       Inhibits protein synthesis                         Anaerobes

  Vancomycin (n=6)      0.625 mg/ml   Ad libitum in drinking water, Days -6 to -1   Glycopeptide      Inhibits peptidoglycan synthesis                   Gram +

  Streptomycin (n=8)    5.0 mg/ml     Ad libitum in drinking water, Days -6 to -1   Aminoglycoside    Inhibits protein synthesis                         Gram +/-

  Cefoperazone (n=4)    0.5 mg/ml     Ad libitum in drinking water, Days -6 to -1   Cephalosporin     Inhibits peptidoglycan synthesis                   Gram +/-

  Ampicillin (n=6)      0.5 mg/ml     Ad libitum in drinking water, Days -6 to -1   Penicillin        Inhibits peptidoglycan synthesis                   Gram +/-

  Metronidazole (n=7)   1.0 mg/ml     Ad libitum in drinking water, Days -6 to -1   Nitromidazole     Incorporates into bacterial DNA, making unstable   Anaerobes
  ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------

**Table 1. Description of Antibiotics used in this study.** The number
of animals per group (n) are shown for each treatment. Days of
antibiotic administration are relative to the day of *C. difficile*
challenge or Day 0.

**\
SUPPLEMENTARY INFORMATION**

[graph]

**Supplemental Figure 1.** correlation results of inv simpson. Inverse
Simpson (**Supplemental Figure 1**) and observed a significant negative
correlation with subsequent *C. difficile* colonization (both p\<0.001).

![](media/image11.pdf)![](media/image12.pdf)

**Supplemental Figure 2.** Shows the adjusted R^2^ and the BIC values
for models tested.

  ----------------------------------------------------------------------------------------------------------------------------
  **Antibiotic**   **Administration**                            **Original Dose **   **Titration Doses**
                                                                                      
                   **(Route, Time)**                                                  
                                                                                      
  ---------------- --------------------------------------------- -------------------- --------------------- ------------------
  Cefoperazone     Ad libitum in drinking water, Days -6 to -1   0.5 mg/ml (n=4)      0.3 mg/ml (n=12)

  Streptomycin     Ad libitum in drinking water, Days -6 to -1   5.0 mg/ml (n=8)      0.5 mg/ml (n=9)

  Vancomycin       Ad libitum in drinking water, Days -6 to -1   0.625 mg/ml (n=6)    0.3 mg/ml (n=8)
  ----------------------------------------------------------------------------------------------------------------------------

**Supplemental Table 1. Titration amounts for each antibiotic.** The
number of animals per group (n) are shown for each antibiotic and dose.
Days of antibiotic administration are relative to the day of *C.
difficile* challenge or Day 0.

**[TABLE]**

**Supplemental Table 2. Description of the 19 candidate OTUs for
consideration in the model. **

